projects/uitdatabank/docs/search-api/filters/datetime.md

```diff
@@ -1,14 +1,247 @@
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+Any dateFrom and dateTo date in ISO-8601.
+
+**Examples**
+
+Get all events available between January 1st, 2025 and March 31, 2025:
+
+```
+GET /events/?q=availableRange:[2025-01-01T00\:00\:00%2B01\:00 TO 2025-03-31T23\:59\:59%2B01\:00]&availableFrom=*&availableTo=*
+```
+
+## Filtering on calendarType
+
+Every event and place in UiTdatabank has one of the following four calendarTypes:
+* `single`: the event occurs on a single date, indicated by a single startDate and endDate.
+* `multiple`: the event occurs on multiple dates, and has multiple subEvent entries with each a different startDate and endDate.
+* `periodic`: the event or place runs for a specific period as indicated by its startDate and endDate, and can optionally have openingHours.
+* `permanent`: the event or place is permanent and has no startDate or endDate, but it can optionally have openingHours.
+
+With the `calendarType` field you can look for results that match a certain calendarType.
+
+`calendarType` is available as both an URL parameter and as an advanced query field:
+* URL parameter: `calendarType`
+* Advanced query parameter: `calendarType`
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+* events: `single` `multiple` `periodic` `permanent`
+* places: `periodic` `permanent`
+
+**Examples**
+
+Retrieve all permanent events and places using the advanced query parameter:
+
+```
+GET /offers/?q=calendarType:permanent
+```
+
+Or alternatively, you can use the URL parameter which will give the exact same results:
+
+```
+GET /offers/?calendarType=permanent
+```
+
+## Filtering on creation date
+
+UiTdatabank keeps track of the creation dates of each document. This metadata for the creation date can be queried with the following parameters:
+
+**URL-parameters**
+* `createdFrom`: returns only results that are created after the given date-time. 
+* `createdTo`: returns only results that created up until the given date-time. 
+
+**Advanced query fields**
+* `created`: returns results that are created in a gived date range.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` 
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+> Note that the + sign must be encoded for URLs (as %2B). Otherwise, it will be interpreted as whitespace and the given date time will be considered invalid.
+
+**Example**
+
+Get all events and places that are created since the start of 2022:
+
+```
+GET /offers/?q=created:[2022-01-01T00\:00\:00%2B01\:00 TO *]
+```
+
+Get all places created before 2024:
+
+```
+GET /places/?createdTo=2024-01-01T00\:00\:00%2B01\:00
+```
+
+## Filtering on date
+
+You can filter on the date that an event is happening or place is open for visits.
+
+**URL-parameters**
+* `dateFrom`: returns only results that are happening / openafter the given date-time. 
```

Comment:
* `dateFrom`: returns only results that are happening / open after the given date-time.